backend/prisma/schema.prisma

```diff
@@ -43,13 +43,13 @@ model Transaction {
 }
 
 model Budget {
-  id        String   @id @default(cuid())
-  userId    String
-  amount    Float
-  weekStart DateTime
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  user      User     @relation(fields: [userId], references: [id])
+  id         String   @id @default(cuid())
+  userId     String
+  amount     Float
+  monthStart DateTime?
+  createdAt  DateTime @default(now())
+  updatedAt  DateTime @updatedAt
+  user       User     @relation(fields: [userId], references: [id])
```

Comment:
probably consider adding a unique constraint on `userId` and `monthStart` to prevent duplicate budgets for the same month?